lang/erlang/Makefile

```diff
@@ -296,26 +296,15 @@ $(call Package/erlang/Default/description)
  This Erlang/OTP package provides functions for exporting XML data to an external format
 endef
 
-# Host
-# host-compile is done with LibreSSL provided by OpenWrt tools/libressl
-
-HOST_CFLAGS += \
-        -DHAS_EVP_PKEY_CTX \
-        -DHAVE_EVP_CIPHER_CTX_COPY
-
 HOST_CONFIGURE_ARGS += \
 	--with-ssl="$(STAGING_DIR_HOST)" \
-	--disable-hipe \
-	--disable-pgo \
-	--disable-smp-support \
```

Comment:
This is removed why?

Reply:
I believe CFLAGS modification was required for older versions of erlang/libressl. Latest Erlang builds and works without it.
Regarding removed configure options - there are no such switches in Erlang anymore. HiPE was completely removed, VM always builds in SMP mode, and `--disable-pgo` has never existed I think.

Reply:
Soo about CLAFGS, shouldnt you do it in separate commit, so it is easier to revert it, if anything happens?

Reply:
Ok, but you should mention that in the commit description, because from the commit, it looks like removing, but why? Because we can. :D

Reply:
Changed  "remove CFLAGS modification required for older versions" into "remove CFLAGS modification since it has been fixed in the upstream", hope it looks better for you. Here is the fix in the upstream - https://github.com/erlang/otp/commit/8e5170d79e387d7e898d948a78e0c70407a4030c.